docs/app/pages/css/css-sections/buttons/colored-buttons/colored-buttons.component.html

```diff
@@ -18,6 +18,10 @@
 
 <hr>
 
+<p *ngIf="theme === SiteThemeId.MicroFocusNext">
```

Comment:
Just one suggestion, could you put this inside an alert component, like we did [here](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/input-controls#flat-text-input) (switch to 2021 theme).

Reply:
I forgot to mention, the flat button documentation section also features a neutral button.